Content.Server/GameObjects/Components/Mobs/Speech/owoAccentComponent.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Content.Server.GameObjects.Components.Mobs.Speech
+{
+    [RegisterComponent]
+    public class OwOAccentComponent : Component, IAccentComponent
```

Comment:
The name of this file should be `OwOAccentComponent.cs`, not `owoAccentComponent.cs` as it is now.